backend/api/services/CreditTradeService.py

```diff
@@ -112,7 +115,13 @@ def create_history(credit_trade, is_new=False):
         history.save()
 
     @staticmethod
-    def approve(credit_trade):
+    def approve(credit_trade, user):
+        """Only government users can approve transfers"""
+        gov_org_type = OrganizationType.objects.get(type="Government")
+
+        if user.organization.type_id != gov_org_type.id:
```

Comment:
I think this should be happening on the view/api level, not in this helper function. The API should be responsible for enforcing the contract. I think it should also rather make use of the built in django permissions, then it can be enforced by adding a decorator on the endpoint.